Update Installation-Troubleshooting.rst (#3916)
ros2/ros2_documentation: source/How-To-Guides/Installation-Troubleshooting.rst

Although it is usually installed by default in Windows 10, we know that some Windows 10-like versions don't have it installed by default (e.g.: Windows Server 2019).
In case you don't have it installed, you can download it from `here <https://www.microsoft.com/en-us/download/details.aspx?id=40784>`_.

Failed to create process
^^^^^^^^^^^^^^^^^^^^^^^^

If running a ROS binary gives the error:

> | failed to create process.

Reviewer: ```suggestion
.. code-block::

   | failed to create process.
```

It is likely the Python interpreter was not found.

> For each executable, the shebang (first line) of the accompanying script is used, so make sure Python is available under the expected path (default: `C:\Python38\`).

Reviewer: ```suggestion
For each executable, the shebang (first line) of the accompanying script is used, so make sure Python is available under the expected path (default: ``C:\Python38\``).
```

Although it is usually installed by default in Windows 10, we know that some Windows 10-like versions don't have it installed by default (e.g.: Windows Server 2019).
In case you don't have it installed, you can download it from `here <https://www.microsoft.com/en-us/download/details.aspx?id=40784>`_.

Failed to create process
^^^^^^^^^^^^^^^^^^^^^^^^

If running a ROS binary gives the error:

| failed to create process.

It is likely the Python interpreter was not found.

> For each executable, the shebang (first line) of the accompanying script is used.
> Make sure Python is available under the expected path (default: `C:\Python38\`) or modify the script files with a correct shebang.

Reviewer: ```suggestion
Make sure Python is available under the expected path (``C:\Python38\``) which is the default when installing Python from chocolatey.
```

Author: On my system I needed a different Python 3.8.x installed, so I simply could not complete the chocolatey default install. I think putting forward an alternative from the default install is useful, what do you think?

Reviewer: The thing is that I don't really think it is reliable advice.  You'd have to change every single file under `Scripts` to make it work, and that just doesn't seem viable.  To me, the best thing we can do here is to advise users to either copy or link wherever they have Python installed to the appropriate place.  So with that said, maybe we can also add the line:

```
If you have Python installed to another location, please copy or link it to ``C:\Python38\``.
```

Author: I agree changing the files should not be the recommended approach. 

However, I would urge we mention the _why_ behind the Python path somewhere, for the rare case where it is just not possible to have the default path or a `venv` is to be used.

We can phrase it like:

- Python must match the script shebang
- So reinstall Python or create a link

With this information an advanced user can solve it their own way and normal users will take the recommended approach.
